src/js/api/set-config.js

```diff
@@ -0,0 +1,71 @@
+define([
+    'utils/underscore',
+], function(_) {
+
+    const supportedFields = [
+        'repeat',
+        'volume',
+        'mute',
+        'autostart'
+    ];
+    let model;
+
+    function setVolume(mute, volume) {
+        const muteSet = !_.isUndefined(mute);
+        const volumeSet = !_.isUndefined(volume);
+
+        if (volumeSet) {
+            model.setVolume(volume);
+            if (!muteSet) {
+                model.setMute(false);
+            }
+        }
+
+        if (muteSet) {
+            model.setMute(mute);
+        }
+    }
+
+    function setAutostart(controller, autoStart) {
+        model.set('autostart', autoStart);
+
+        const autoStartOnMobile = model.autoStartOnMobile();
+        if (autoStartOnMobile) {
+            model.set('autostartMuted', true);
+        }
+        model.set('playOnViewable', autoStartOnMobile || autoStart === 'viewable');
```

Comment:
Since this is lifted directly from https://github.com/jwplayer/jwplayer/blob/master/src/js/controller/setup-steps.js#L294 I think it would be better to break both into a single method